src/test/java/net/mcreator/integration/workspace/ReferencesFinderTest.java

```diff
@@ -104,6 +91,9 @@ public class ReferencesFinderTest {
 				TestWorkspaceDataProvider.getModElementExamplesFor(workspace, type, false, random).forEach(e -> {
 					workspace.addModElement(e.getModElement());
 					workspace.getModElementManager().storeModElement(e);
+					LocalizationUtils.generateLocalizationKeys(workspace.getGenerator(), e,
+							(List<?>) generatorConfiguration.getDefinitionsProvider()
+									.getModElementDefinition(e.getModElement().getType()).get("localizationkeys"));

```

Comment:
Forbidden?

Reply:
Just add comment why this is here so someone looking to this code for the first time can understand your intent

Reply:
:heavy_check_mark: Fixed